src/common/utils/common.filter.utils.ts

```diff
@@ -37,3 +37,33 @@ export const applyFilterMapping = (filter: Filter, filterMappings: FilterMapping
   }
   return filterConditions
 }
+
+/**
+ * Add not null to filter mapping. Useful if you have sort conditions and you don't want to
+ * include null values.
+ * @param allFilters Filter object
+ * @param filtersNotToBeNull Filters not to be null
+ * @returns Filter object with specified filters not to be null
+ * @example
+ * const allFilters = {
+ * title: 'Book Title',
+ * authorId: 1,
+ * }
+ * const filtersNotToBeNull = ['ratingCount']
+ * const filters = addNotNullToFilterMapping(allFilters, filtersNotToBeNull)
```

Comment:
[SUG] I think it should be this instead, if I'm understanding well
```suggestion
 * const filters = excludeNullValuesFromFilters(allFilters, filtersNotToBeNull)
```